content/en/blog/_posts/2023-11-1-Gateway-API-Future.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: blog
+title: "New Experimental Features in Gateway API v1.0"
+date: 2023-11-02T10:00:00-08:00
+slug: gateway-api-ga
+---
+
+***Authors:*** Candace Holman (Red Hat), Dave Protasowski (VMware), Gaurav K Ghildiyal (Google), John Howard (Google), Simone Rodigari (IBM)
+
+Recently, the [Gateway API](https://gateway-api.sigs.k8s.io/) [announced its v1.0 GA release](/blog/2023/10/31/gateway-api-ga/), marking a huge milestone for the project.
+
+Along with stabilizing some of the core functionality in the API, a number of exciting new *experimental* functionality has been added.
+
+## Backend TLS Policy
+
+`BackendTLSPolicy` is a new Gateway API type used for specifying the TLS configuration of the connection from the Gateway to backend Pods via the Service API object.
+It is specified as a [Direct PolicyAttachment](https://gateway-api.sigs.k8s.io/geps/gep-713/#direct-policy-attachment) without defaults or overrides, applied to a Service that accesses a backend, where the BackendTLSPolicy resides in the same namespace as the Service to which it is applied.
+One can use the same BackendTLSPolicy for all the different Routes that might point to the referenced Service.
```

Comment:
I got feedback on similar content in https://github.com/kubernetes-sigs/gateway-api/pull/2547, this is preferred:
```suggestion
All Gateway API Routes that point to a referenced Service should respect a configured BackendTLSPolicy.
```